src/Dashboard/components/Dashboard/Dashboard.tsx

```diff
@@ -51,52 +58,92 @@ export default function Dasxboard() {
                   <div className="tasks d-flex flex-column  p-3  rounded-4">
                     <i className="fa fa-tasks my-3"> </i>
                     <span className="py-3 text-dark-light">To do</span>
-                    <h3 className="text-dark-light">{"daa"}</h3>
+                    <h3 className="text-dark-light">{todoCount}</h3>
                   </div>
                 </div>
                 <div className="col-md-4 ">
                   <div className="tasks1 d-flex flex-column  p-3 rounded-4">
-                  <i className="fa-solid fa-group-arrows-rotate  my-3"></i>
+                    <i className="fa-solid fa-group-arrows-rotate  my-3"></i>
                     <span className="py-3 text-dark-light">In-progress</span>
-                    <h3 className="text-dark-light">{"ada"}</h3>
+                    <h3 className="text-dark-light">{progressCount}</h3>
                   </div>
                 </div>
                 <div className="col-md-4">
                   <div className="tasks2 d-flex flex-column   p-3  rounded-4">
-                  <i className="fa-solid fa-calendar-check my-3"></i>
+                    <i className="fa-solid fa-calendar-check my-3"></i>
                     <span className="py-3 text-dark-light">Done</span>
-                    <h3 className="text-dark-light">{"ada"}</h3>
+                    <h3 className="text-dark-light">{doneCount}</h3>
                   </div>
                 </div>
               </div>
             </div>
 
-            <div className=" w-50 px-5 ms-4 bg-white rounded-4">
-              <div className="titles">
+            <div className="col-md-5 bg-white rounded-4">
+              <div className=" tasks-charts d-flex justify-content-center pt-5 align-items-center">      
+            <DonutChart
+                data={[
+                  { label: "To Do", value: todoCount },
+                  { label: "In Progress", value:progressCount  },
+                  { label: "Done", value: doneCount },
+                ]}
+                height={290}
+                width={400}
+                clickToggle={true}
+                interactive={true}
+                innerRadius= {0.6}
+                // colors = {["#7187FF", "#9D9D9D", "#CFD1EC"]}
+                  colors={["#CFD1EC", "#E4E4BC", "#E7C3D7"]}
+              />
+                 </div>
+            </div>
+          </div>
+        </div>
+        <div className="container-fluid py-5">
```

Comment:
The Users charts should be rendered conditionally, for employees it should be removed, since they don't have access to the API.